python/ql/test/experimental/query-tests/Security/CWE-295/urlopen.py

```diff
@@ -0,0 +1,26 @@
+import urllib.request
+
+#Simple cases
+urllib.request.urlopen('https://semmle.com', "/path/to/cert/") # BAD (does not assign cert path to "cafile")
+urllib.request.urlopen('https://semmle.com', cafile="/path/to/cert/")
```

Comment:
(This one should be marked `# OK`, I assume.)

Also could we get an example that sets `cafile` to something appropriate, and `capath` to `None` (to check that we're handling this case correctly, and not counting it as insecure)?